libs/plugins/logrocket/src/lib/plugins-logrocket.ts

```diff
@@ -0,0 +1,25 @@
+import { registerPlugin, getMyConfig, HandledRoute } from '@scullyio/scully';
+
+export const LogRocket = 'logrocket';
+
+const logrocketPlugin = async (
+  html: string,
+  route: HandledRoute
+): Promise<string> => {
+  const logrocketConfig = getMyConfig('logrocket');
```

Comment:
```suggestion
  const logrocketConfig = getMyConfig(logrocketPlugin);
```
Works better with just the plugin itself

Reply:
I noticed you used the plugin itself in the bas-href-rewrite, but I was not sure how to do it. Thx @SanderElias!

Reply:
I think more than a default config, it should throw an error if there is no config. I'll make the changes